src/cascadia/TerminalSettingsEditor/GlobalAppearance.cpp

```diff
@@ -16,6 +20,19 @@ using namespace winrt::Windows::Foundation::Collections;
 
 namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 {
+    // For ComboBox an empty SelectedItem string denotes no selection.
+    // What we want instead is for "Use system language" to be selected by default.
+    // --> "und" is synonyme for "Use system language".
+    constexpr std::wstring_view systemLanguageTag{ L"und" };
+
+    static bool starts_with(const std::wstring_view str, const std::wstring_view prefix)
+    {
+#ifdef __cpp_lib_starts_ends_with
+#error This code can be replaced in C++20, which natively supports .starts_with().
+#endif
+        return str.size() >= prefix.size() && std::wstring_view::traits_type::compare(str.data(), prefix.data(), prefix.size()) == 0;
+    };

```

Comment:
I guess this could be put elsewhere as a generic function, but I wasn't sure where it would be appropriate.

Reply:
probably just `til`

Reply:
we have a til header that contains visualize_control_codes, which is also string work